test/helpers.jl

```diff
@@ -1,5 +1,17 @@
 # Utility functions for testing
 
+if VERSION < v"1.9.0-"  # https://github.com/JuliaLang/julia/pull/47367
+    macro allocations(ex)
+        quote
+            Base.Experimental.@force_compile
```

Comment:
CI failures suggest that Julia 1.6 does not know `Base.Experimental.@force_compile`

Reply:
ah, thanks! in which case we can switch to `while false; end` (https://github.com/JuliaLang/julia/pull/42128/)
```suggestion
            while false; end  # want to force compilation, but v1.6 doesn't have `@force_compile`
```